README.md

```diff
@@ -21,11 +17,7 @@ This example is deployed using [Netlify](https://docs.lowdefy.com/deployment).
     - You can read more about the [Lowdefy MongoDB connector](https://docs.lowdefy.com/MongoDB).
 - Clone this repository.
 - Create a `.env` file in your project folder and set your MongoDB database connector URI as a variable in the `.env` file: `LOWDEFY_SECRET_EXAMPLES_MDB="{{ your_mongodb_connection_uri }}"`
-- In the command console, navigate to your project folder and run the Lowdefy CLI: `npx lowdefy@latest dev`.
-
-## Deploying this example
-
-This simplest solution is to deploy this example to [Netlify](https://netlify.com). See [the Netlify Deployment instructions](https://docs.lowdefy.com/deployment) for more detail on how to deploy a Lowdefy app to Netlify.
+- In the command console, navigate to your project folder and run the Lowdefy CLI: `pnpx lowdefy@4.0.0-alpha.37 dev`.
```

Comment:
Lets rather say `pnpx lowdefy@4 dev`. This won't work until we release v4 stable, but then we don't need to come back and fix it

Reply:
Change made in 49e2cf579f5a19308fb9eb9be180d4f5483fd24a